lib/eunit/src/eunit_data.erl

```diff
@@ -131,8 +132,8 @@ iter_id(#iter{pos = N, parent = Ns}) ->
 
 %% @spec (testIterator()) -> none | {testItem(), testIterator()}
 
-iter_next(I = #iter{next = []}) ->
-    case next(I#iter.tests) of
+iter_next(I = #iter{next = [], options = Options}) ->
```

Comment:
How will this work? `iter_next/1` is exported in the beginning of the file but after this change there is no `iter_next/1` anymore

Reply:
It looks to me that it still only has one argument, the map!

Reply:
It's a new field inside map, arity of function is unchanged I think.